src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+- **TODO**. Ideas: sensor sweep? Mass applying (un)bolting to everything an APC’s connected to? Atmospherics reset action that can be used from the action bar/alt-click radial menu and applied to an atmos alarm to reset it to default? Clicking vents to unlock them?
+- Consoles! They aren’t free and cut a decent chunk, to encourage the AI player to choose a Compute option, but in a pinch this should work too. It also gets paid by the player having to manually locate a console and being unable to check on things elsewhere but the vicinity of the console.
+
+### Sources
+
+- AI specialized machinery, contributing a significant amount of processing capability to the AI.
+  - The downside to this is that it requires someone physical to build it, and needs a decent amount of resources
+  - Such machinery should be difficult to destroy or even move. It stays where it stays, unless the situation requires it and someone puts enough effort to move it; or break it.
+  - Generates heat up to a throttle cutoff, and works better when cooled. When not in an optimal environment, provides less computing power, but still comparable to servers.
+  - Backup power? Camera built in?
+- Crew devices; computers, machines, servers
+  - These don’t contribute as much **Compute** as they are not specialized for AI use. Servers do provide more **Compute** than just consoles, and provide **Bandwidth** in amounts comparable or identical to dedicated machines..
+  - Locks out the device for crew use, rendering it inoperable. This acts as a conflict point with the crew. Computers don’t work, machines either. Servers don’t provide normal functionality or do so in a very limited way.
+    - E.g. crew monitoring server could give invalid/partial data with a warning, or just not work at all. Same for telecom servers.
+  - Non-antag AI can be flushed out from the device via a near-instant crew exposed action. This includes a subverted AI. Could be an UI that shows “This device is in use by AI \<name\>. \<Override (button)\>.”
+  - Such shooing away should keep the AI from taking the device over for at least some time, at least a minute. It should not be able to just annoy the crew to death even when compromised - it’s still shackled.
+  - Antagonist AI, “Malf”, should have an option or be able to spend some resources to completely lock the crew out. Will require more detailed balancing later.
+  - For “Malf”, the timeout should be reduced or bypassable. Again, might need some balance tweaking whether it’s just a pop up “you sure?” or a resource sink.
+- Borgs! Useful and somewhat plentiful, perfect for an emergency ~~snack~~ method of survival when the station’s powered off and the AI SMES is about to drain, with a downside of rendering the borg unable to synthesize voice (speak) when AI’s along for the ride, except for beeps and emotes. Does not require borg sync, but requires approval from the borg even if synced. Malf AI can bypass the approval.
```

Comment:
This needs to be fleshed out/explained further cause I don't know the details of this, it's hard to follow.

So an AI can hop in a borg's body with consent from the Borg player (or just automatically if there's a Posi brain waiting for a ghost), but the AI can't speak, only beep/emote. And it doesn't require sync.

Should this control require an obscene bits/s? This is giving AI a pretty direct way of control over the physical world.

When a Malf AI hacks a borg, does that force a sync? Does it appear synced to crew, or will it cause a fake sync?
For the purposes of metagaming, can a borg choose to sync/unsync itself at will, or will it require crew intervention? 

Probably the latter, as it makes it good for lowpop and metagaming. A borg should require crew intervention to unsync if synced to a Malf AI, but otherwise, it can unsync itself at will (if it would ever want to).

Reply:
This is not a way of *controlling* a borg's body at all, this is specifically to use them as Compute generating entity. I don't have any plans for letting AI just have a physical body, this defeats entire point of this proposal. Any physical body providing thing would basically cut the AI off its network and give you effectively yet another borg.

Good points on borg sync though.

Reply:
Ahhhhhh, I see now, that makes much more sense.